starter-code/package.json

```diff
@@ -3,20 +3,31 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "nodemon ./bin/www"
+    "dev": "nodemon ./bin/www"
```

Comment:
Better add at least two scripts, one for start (with node) and another for development (with nodemon).